### ISSUE TITLE
All folders creation date is 01.01.1970, 05:00:00

### ISSUE THREAD

**Issue by HalfBottleOfMind** [UniSharp/laravel-filemanager#869]
Windows 10, php artisan serve.
![image](https://user-images.githubusercontent.com/15110331/72344765-dcf1ac00-36f3-11ea-8ea6-c901873084c0.png)

**Comment by dissolvdgrl** [comment]
Same issue.

**Comment by mnvx** [comment]
Same in ubuntu 18.04

**Comment by mnvx** [comment]
I has added
```js
console.log(item.time);
```
Before this line
https://github.com/UniSharp/laravel-filemanager/blob/a2c390b1414fa35457fc5c6d01068dcf6dac7cd2/public/js/script.js#L461

And I see what `item.time` is timestamp for files and `false` for folders.

**Comment by mnvx** [comment]
I has tried to replace these lines 

https://github.com/UniSharp/laravel-filemanager/blob/master/src/LfmItem.php#L109-L116

to

```php
    public function time()
    {
        return $this->lfm->lastModified();
    }
```

And it works well.

**Comment by mafftor** [comment]
> I has tried to replace these lines
> 
> https://github.com/UniSharp/laravel-filemanager/blob/master/src/LfmItem.php#L109-L116
> 
> to
> 
> ```
>     public function time()
>     {
>         return $this->lfm->lastModified();
>     }
> ```
> 
> And it works well.

Yes, this is the rightest solution, use my version of this package, i've changed it
https://github.com/mafftor/laravel-file-manager
